tests/repository/booking_queries_test.py

```diff
@@ -151,12 +162,36 @@ def test_returns_only_used_bookings(self, app: fixture):
         repository.save(thing_booking1, thing_booking2)
 
         # When
-        bookings = booking_queries.find_eligible_bookings_for_offerer(offerer.id)
+        bookings = booking_queries.find_payment_eligible_bookings_for_offerer(offerer.id)
 
         # Then
         assert len(bookings) == 1
         assert thing_booking1 in bookings
 
+    @pytest.mark.usefixtures("db_session")
+    def test_returns_only_past_event_bookings(self, app: fixture):
+        # Given
+        user = create_user()
```

Comment:
```suggestion
        pro = create_user()
```
Suite à la discussion que l'on a eu.

Reply:
C'est un usager (jeune) plutôt.

Reply:
`beneficiary` alors :P

Reply:
Oui, c'est le terme que j'ai retenu :)